P5/Source/Specs/value.xml

```diff
@@ -25,6 +27,15 @@ $Id$
   <content>
     <macroRef key="macro.xtext"/>
   </content>
+  <!-- TODO Phase 3: Mention conversion script as part of deprecation warning. #1805 -->
+  <constraintSpec scheme="schematron" validUntil="2021-07-09"
```

Comment:
Delete entire `<constraintSpec>`. The required Schematron will (or at least should) be generated by the build process based on the `@validUntil` and `<desc type=deprecationInfo>` of the `<elementSpec>`.